WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListViewModel.swift

```diff
@@ -5,7 +5,7 @@ import Yosemite
 
 final class CustomFieldsListViewModel: ObservableObject {
     private let stores: StoresManager
-    @Published private var originalCustomFields: [CustomFieldViewModel]
+    @Published private(set) var originalCustomFields: [CustomFieldViewModel]
```

Comment:
Was this an accidental change to make this field public?

Reply:
Oh, this was set to public because I initially had `disallowedKeysForCreation` logic in the View, thanks for noticing. I'll set it back.